docs/src/faq.md

```diff
@@ -0,0 +1,24 @@
+# FAQ
+
+## Why are the RGB colors encoded in the `ColorMixture` *type*? Why not a value field?
+
+In many places, JuliaImages assumes that you can convert from one color space to another purely from knowing the type you want to convert to. This would not be possible if the RGB colors were encoded as a second field of the color.
```

Comment:
```suggestion
In many places, JuliaImages assumes that you can convert from one color space to another purely from knowing the type you want to convert to. This would not be possible if the RGB colors were encoded as a second field of the color.

If you consider an entire image `Array{<:ColorMixture}`, the mental model JuliaImages uses, then you'll realize that storing the RGB colors as a value field would also hurt the performance because it requires an additional set of memories for each pixel.
```